js/components/graph/Node.js

```diff
@@ -33,18 +33,21 @@ export default function Node(props) {
 
   let ellipse = null
   const newClassName = props.className + " " + props.status
-  if (props.highlighted) {
+  if (props.highlightFocus || props.highlightDeps) {
     const attrs = props.JSON
     const width = parseFloat(attrs.width) / 2
     const height = parseFloat(attrs.height) / 2
     ellipse = (
-      <ellipse
-        className="spotlight"
-        cx={parseFloat(attrs.pos[0]) + width}
-        cy={parseFloat(attrs.pos[1]) + height}
-        rx={width + 9}
-        ry={height + 8.5}
-      />
+      <svg>
```

Comment:
Okay so it's good that you moved the `def`, but also you can remove the `<svg>` entirely and keep the `<ellipse>` (following the original version of the code); the only part that should be showing up in the diff is the new `filter` attribute.